shared/src/main/scala/squants/motion/AngularAcceleration.scala

```diff
@@ -3,7 +3,7 @@ package squants.motion
 import squants.mass.MomentOfInertia
 import squants.space._
 import squants.time.{Seconds, Time, TimeDerivative}
-import squants.{AbstractQuantityNumeric, Dimension, Length, PrimaryUnit, Quantity, SiUnit, UnitConverter, UnitOfMeasure}
+import squants.{AbstractQuantityNumeric, Dimension, Length, PrimaryUnit, Quantity, SiUnit, SquantifiedDouble, UnitConverter, UnitOfMeasure}
```

Comment:
`SquantifiedDouble` is an implicit class that triggers somewhere in this file and I'm surprised that it compiled under Scala 2 (it needs to be in scope).